jolly-jellyfish/src/page_maker/migrations/0001_initial.py

```diff
@@ -1,12 +1,15 @@
-# Generated by Django 3.0.8 on 2020-08-01 12:33
+# Generated by Django 3.0.9 on 2020-08-03 15:27
```

Comment:
[Django aims to be as backward compatible as possible](https://docs.djangoproject.com/en/3.1/misc/api-stability/), I don't think it will be too much of an issue